net-mgmt/pfSense-pkg-arpwatch/files/usr/local/pkg/arpwatch.inc

```diff
@@ -228,7 +232,7 @@ $fd = fopen('php://stdin','r');
 $message = stream_get_contents($fd);
 fclose($fd);
 
-if (false !== $message) {
+if ((false !== $message) && (false === strpos($message, ': Cron '))) {
```

Comment:
Should this be done unconditionally in this way? The feature request mentioned adding a checkbox/option for it.

Reply:
Checkbox added, but this is weird behavior anyway
https://redmine.pfsense.org/issues/8454

Reply:
Unfortunately, since arpwatch can only use `/usr/sbin/sendmail` to send mail, anything else that wants `/usr/sbin/sendmail` will also take advantage of it being there (Notably Cron). There isn't really a good way around that as far as I'm aware. Unless the script can check something in its environment to tell it's being called by arpwatch specifically and nothing else.

Reply:
it should be OK with this `Disable Cron emails` checkbox for now